app/src/main/java/com/infomaniak/drive/data/models/UISettings.kt

```diff
@@ -33,6 +33,23 @@ class UISettings(val context: Context) {
         }
     }
 
+    fun getSaveExternalFilesPref(): Triple<Int, Int, Int> {
```

Comment:
I would use a Bundle or a data-object instead of a Triple, but anyway it's correct !